pkgs/tools/cd-dvd/srt-to-vtt-cl/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, which, substituteAll }:
+
+stdenv.mkDerivation rec {
+  pname = "srt-to-vtt-cl-unstable";
+  version = "2019-01-03";
```

in 2 comments cause i can't figure this stuff out
```suggestion
  version = "unstable-2019-01-03";
```